sources/locales/ru.js

```diff
@@ -0,0 +1,79 @@
+export default {
+	Contacts: "Контакты",
+	Activities: "Действия",
+	Settings: "Настройки",
+	"Type something": "Напишите что-нибудь",
+	Delete: "Удалить",
+	Edit: "Редактировать",
+	Files: "Файлы",
+	State: "Статус",
+	Type: "Тип",
+	DueDate: "Дата",
+	Details: "Детали",
+	"Add Activity": "Добавить действие",
```

Comment:
Make sure all elements and their labels look appropriate:
![image](https://user-images.githubusercontent.com/7587383/77418850-8af69200-6dd8-11ea-9200-ba44d805386f.png)